using_images/sti_images/perl.adoc

```diff
@@ -10,10 +10,18 @@
 toc::[]
 
 == Overview
-OpenShift provides https://github.com/openshift/source-to-image[STI] enabled Perl images for building and running Perl applications.  These images can be used with the link:../../architecture/core_objects/builds.html#sti-build[STI build strategy].
+OpenShift provides https://github.com/openshift/source-to-image[STI] enabled
+Perl images for building and running Perl applications. These images can be used
+with the
+link:../../architecture/core_objects/builds.html#sti-build[STI build strategy];
```

Comment:
@jhadvig -- No. :) The semicolon indicates that what follows is a continuation of the sentence. A semicolon is used between two closely related independent clauses.